AEPCore/Tests/FunctionalTests/Configuration/ConfigurationLifecycleResponseTests.swift

```diff
@@ -80,36 +80,4 @@ class ConfigurationLifecycleResponseTests: XCTestCase {
         mockSystemInfoService.property = nil
     }
 
-    /// Tests that app id is loaded from manifest on normal startup after being persisted
-    func testHandleValidAppidFromManifestPersisted() {
```

Comment:
should we keep this test?

Reply:
I don't believe so. The first part of the test ensures that the app is loaded from the manifest, this part is still valid. But the second half of the tests removes the app if from the manifest and checks if the app is is loaded from persistence which is no longer valid. 

Maybe we can keep the first half of the test? I'd assume we already have a test for that case.